nettacker/api/engine.py

```
@@ -383,16 +384,16 @@ def get_result_content():
     scan_id = get_value(flask_request, "id")
     if not scan_id:
         return jsonify(structure(status="error", msg=_("invalid_scan_id"))), 400
-
+    
     try:
         filename, file_content = get_scan_result(scan_id)
     except Exception:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)